src/utils/apis/index.ts

```diff
@@ -0,0 +1,22 @@
+import * as _axios from 'axios';
+
+export const axios = _axios.default.create({
+  baseURL: '/api',
+});
+
+export const authAxios = _axios.default.create({
+  baseURL: '/api',
+});
+
+authAxios.interceptors.request.use(
+  (config) => {
+    const accessToken = localStorage.getItem('accessToken');
+    if (accessToken) {
+      config.headers.Authorization = 'Bearer ${accessToken}';
```

Comment:
p1) 토큰 설정 문자열을 '로 사용한 이유가 있나요? 특별한 이유가 없다면 템플릿 리터럴 사용할 때 백틱(`)을 사용해주세요!!
```suggestion
      config.headers.Authorization = `Bearer ${accessToken}`;
```

Reply:
헐 이런 걸 캐치해주시다니! 가문의 영광임둥! 오타였슴둥!